examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -86,7 +86,7 @@ async function enterCreditCardDetailsAndContinue(page: Page): Promise<void> {
   await stripeFrame.getByPlaceholder("CVC").fill("123");
   await stripeFrame.getByLabel("Country").selectOption("US");
   await stripeFrame.getByPlaceholder("12345").fill("12345");
-  await page.getByRole("button", { name: "Pay" }).click();
+  await page.getByRole("button", { id: "Pay" }).click();
```

Comment:
From looking at Playwright's documentation, not sure if we can reference an id directly :/. I did see you can reference by `data-testid`: https://playwright.dev/docs/locators#locate-by-test-id.

Lmk if you would like some help on this!

Reply:
ha I was just lazy (and had to jump on a call). I will make another attempt at fixing and also verify locally